### PR TITLE
PANDARIA: bump up rancher api version

### PR DESCRIPTION
**Pull request by JacieChao**
https://github.com/cnrancher/pandaria/issues/2830

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -99,14 +101,14 @@ require (
...
-	github.com/onsi/gomega v1.24.1 // indirect
+	github.com/onsi/gomega v1.26.0 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
 	github.com/prometheus/client_golang v1.12.1 // indirect
 	github.com/prometheus/client_model v0.2.0 // indirect
 	github.com/prometheus/common v0.32.1 // indirect
 	github.com/prometheus/procfs v0.7.3 // indirect
-	github.com/rancher/aks-operator v1.1.0-rc4 // indirect
+	github.com/rancher/aks-operator v1.1.0-rc5 // indirect
```
Comment (Jason-ZW) [inline go.mod:112]: 是不是要换成cnrancher/aks-operator

File: .drone-pandaria.yml [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    settings:
+      username:
+        from_secret: docker_username
+      password:
+          from_secret: docker_password
+      platforms:
+        - linux/amd64
+        - linux/arm64
+      target: "jacie/rancher-webhook:${DRONE_TAG}"
+      template: "jacie/rancher-webhook:${DRONE_TAG}-ARCH"
```
Comment (Jason-ZW) [inline .drone-pandaria.yml]: 后面统一替换成cnrancher？

**Review by Jason-ZW — CHANGES_REQUESTED** [review]
Others LGTM, 后面final rc后需要更新相关依赖

File: go.mod (hunk 1 above)
Comment (JacieChao) [reply go.mod:112]: done

File: .drone-pandaria.yml (hunk 2 above)
Comment (JacieChao) [reply .drone-pandaria.yml]: done

**Review by jianghang8421 — APPROVED** [review]
LGTM

**Review by Jason-ZW — APPROVED** [review]
LGTM

**Pull request merged**